src/styles/feedback.scss

```diff
@@ -0,0 +1,42 @@
+.feedback {
+  padding-top: var(--amplify-space-xl);
+  overflow: hidden;
+}
+
+.vote-response {
+  flex-direction: column;
+  opacity: 0;
+  overflow: visible;
+  white-space: nowrap;
+  padding-left: 0;
+  font-size: 0.9rem;
+  justify-content: center;
+  gap: 0;
+  pointer-events: none;
```

Comment:
Let's find an accessible way to hide the link. `opacity: 0` and `pointer-events: none` mean this is still accessible to screen readers and keyboard navigation. It needs to not be in the DOM, set to display: none, or maybe visibility: hidden

Reply:
Corrected. Thank you